Extensions/ExtendedVariables-header.json

```diff
@@ -1,15 +1,15 @@
 {
-  "shortDescription": "Conditions and actions to check for the existence of a variable, or delete existing ones from memory.",
+  "shortDescription": "Add some conditions, actions and expressions to check for the existence of a variable, delete existing ones from memory, and create dynamic variables.",
   "extensionNamespace": "",
   "fullName": "Extended variables support",
   "name": "ExtendedVariables",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "url": "Extensions/ExtendedVariables.json",
   "headerUrl": "Extensions/ExtendedVariables-header.json",
-  "tags": "variables, scene variables, global variables, object variables",
+  "tags": "variable,scene,global,object,remove,expression",
   "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/variable-box.svg",
   "eventsBasedBehaviorsCount": 0,
-  "eventsFunctionsCount": 5,
-  "description": "This extension provides conditions to check if a global/scene/object variable exists and actions to delete an existing global/scene variable from memory.",
+  "eventsFunctionsCount": 13,
+  "description": "This extension provides: \n- Conditions to check if a global/scene/object variable exists.\n- Actions to delete an existing global/scene variable from memory.\n- Actions for create a global/scene variable with a text input instead a variable input.\n- Expressions for get the value or text from a global/scene variable with a text input instead a variable input.\n\nText input instead of variable input allow the user to use or create dynamics variable with expressions like: \n`\"MyVar\"+ ToString(Variable(Health))`",
```

Comment:
* "Actions for create a global/scene variable with a text input instead a variable input"
  =>
  Actions to create a global/scene variable using a text expression, rather than a variable name.

  Remember: "Pour" is only translated to "for" for nouns (and verb+ing). If there is a verb, it's always always always "to".

*  "Expressions for get"
  => "Expressions to get" and same fixes

* "Text input instead of variable input allow"
  => Don't forget the "s" to the verb for the 3rd person.

* "dynamics variable"

  Never add a "s" to adjectives in english: dynamic variables.

* But let's avoid speaking about "dynamic" variables, all variables are already dynamic. Let's also avoid talking about "Text Input" because people will confuse it by a widget to enter a text.
* Let's also avoid saying "the user". This description is already read by the user, no need to talk about them.

  Instead: "Text expressions allows to enter the name of the variable with an expression like `\"MyVar\"+ ToString(Variable(Health))`"